dpnp/linalg/dpnp_utils_linalg.py

```diff
@@ -2470,6 +2476,12 @@ def dpnp_qr(a, mode="reduced"):
     )
     _manager.add_event_pair(ht_ev, geqrf_ev)
 
+    # w/a to avoid raice conditional on CUDA during multiple runs
+    # TODO: Remove it ones the OneMath issue is resolved
+    # https://github.com/uxlfoundation/oneMath/issues/626
+    if dpnp.is_cuda_backend(a_sycl_queue):
+        ht_ev.wait()
```

Comment:
Does it still make sense to add the events to the manager list (above call `_manager.add_event_pair(ht_ev, geqrf_ev)`) when we are going make a sync point here in case of a w/a?

Reply:
It is a good point.
I think it makes no sense to add the events to the manager list here
I will update it